airflow-core/tests/unit/cli/commands/test_pool_command.py

```diff
@@ -64,19 +67,28 @@ def test_pool_list_with_args(self):
 
     def test_pool_create(self):
         pool_command.pool_set(self.parser.parse_args(["pools", "set", "foo", "1", "test"]))
-        assert self.session.query(Pool).count() == 2
+        assert self.session.execute(select(func.count()).select_from(Pool)).scalar_one() == 2
 
     def test_pool_update_deferred(self):
         pool_command.pool_set(self.parser.parse_args(["pools", "set", "foo", "1", "test"]))
-        assert self.session.query(Pool).filter(Pool.pool == "foo").first().include_deferred is False
+        assert (
+            self.session.execute(select(Pool).where(Pool.pool == "foo")).scalars().first().include_deferred
```

Comment:
```suggestion
            self.session.scalar(select(Pool).where(Pool.pool == "foo")).include_deferred
```

Same for many others here